Add entitlements to metadata (#13728)
canonical/lxd: doc/reference/index.md

/reference/provided_metrics
```

## Fine-grained permissions

> If you are managing user access via {ref}`fine-grained-authorization`, you can check the available {ref}`permissions <permissions>` that can be assigned to groups.

Reviewer: ```suggestion
If you are managing user access via {ref}`fine-grained-authorization`, check which {ref}`permissions <permissions>` can be assigned to groups.
```

Reviewer: "you can" is a bit misleading since it's totally possible and allowed to check the permissions even if you don't use them. ;)
(Yes, nitpicking here :smile: )
